xc7/primitives/slicem/Ndram/ntemplate.N_dram.pb_type.xml

```diff
@@ -70,22 +70,23 @@
     </metadata>
   </mode>
   <mode name="64_SINGLE_PORT">
```

Comment:
We can probably get rid of the 64_SINGLE_PORT and 32_SINGLE_PORT modes, if we add the correct support for the DI muxes.

Reply:
Done.